.github/workflows/test.yml

```diff
@@ -35,3 +35,5 @@ jobs:
           key: ${{ runner.os }}-sbt-${{ hashFiles('**/*.sbt') }}-${{ hashFiles('**/build.properties') }}
       - name: SBT ci-checks
         run: ./sbtx ci-checks
+        env:
+          CI: false
```

Comment:
???
scalafix and scalafmt should be checked (but not at compile time, at check time)